components/Tabs/src/private/MDCItemBar.m

```diff
@@ -513,7 +523,10 @@ - (void)selectItemAtIndex:(NSUInteger)index animated:(BOOL)animated {
 }
 
 - (UITabBarItem *)itemAtIndexPath:(NSIndexPath *)indexPath {
-  return _items[indexPath.item];
+  if (_items && (NSInteger)_items.count > indexPath.item) {
```

Comment:
Tip: The `items` property is declared `nonnull` so you can remove the check on its existence.